app/lib/search/query_builder.rb

```diff
@@ -56,7 +56,10 @@ def pagination_start
     end
 
     def current_page
-      [params["page"].to_i, 1].max
+      page = Integer(params["page"] || 1)
+      [page, 1].max
+    rescue StandardError => _e
```

Comment:
Think we can remove assigning the exception to a variable (`=> _e`) as it's not in use. It is underscored which implies it's not being used, but we could go one further and remove that part entirely.

Reply:
Ah, I'm so used to old rubocops requiring an assignment I didn't even think to test whether govuk's cop required it. Have removed the assignment. Thanks!